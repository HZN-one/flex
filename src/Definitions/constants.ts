export const HZN_BASE_URL_ASSETS =
  process.env.HZN_BASE_URL_ASSETS ||
  "https://storage.googleapis.com/dev-hzn-assets";

export const DUMMY_IMAGE_URL =
  "https://dummyimage.com/250x180/e5e5e5/e5e5e5.jpg";

export const DUMMY_COMPANY_IMAGE_URL =
  "https://www.freepnglogos.com/uploads/netflix-logo-circle-png-5.png";

export const DEFAULT_IMAGE_INPUT_AVATAR = `${HZN_BASE_URL_ASSETS}/icons/inputAvatar.png`;
