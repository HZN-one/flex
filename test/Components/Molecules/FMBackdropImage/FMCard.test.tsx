/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMBackdropImage } from "@Molecules";

import { render } from "@testing-library/react";

import { DUMMY_COMPANY_IMAGE_URL } from "@Definitions/constants";

describe("FMBackdropImage", () => {
  let open = true;
  const handleClose = () => {
    open = false;
  };
  it("should match snapshot", () => {
    const { container } = render(
      <FMBackdropImage
        testID="backdrop-image-test"
        handleClose={handleClose}
        open={open}
      >
        <img src={DUMMY_COMPANY_IMAGE_URL} />
      </FMBackdropImage>
    );
    expect(container).toMatchSnapshot();
  });
});
