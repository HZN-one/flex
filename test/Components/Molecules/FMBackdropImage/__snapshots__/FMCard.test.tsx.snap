// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMBackdropImage should match snapshot 1`] = `
<div>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root css-sox5kk-MuiBackdrop-root"
    data-testid="backdrop-image-test"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <img
      src="https://www.freepnglogos.com/uploads/netflix-logo-circle-png-5.png"
    />
  </div>
</div>
`;
